app/controllers/evidence_controller.rb

```diff
@@ -108,12 +110,18 @@ def autogenerate_issue
   end
 
   def liquid_resource_assigns
+    # find evidence inline for preview action
+    @evidence = @evidence || @node.evidence.find_by(id: params[:id])
```

Comment:
this is a smell and not #liquid_resource_assigns' task.

Reply:
Removed